modules/mcuboot/flash_sim.conf

```diff
@@ -0,0 +1 @@
+CONFIG_ADD_MCUBOOT_MEDIATE_SIM_FLASH_DTS=y
```

Comment:
This does not look right.

This config depends on CONFIG_BOOTLOADER_MCUBOOT, so it cannot be added to the MCUboot image.
This option was used to add a flash_sim DTS overlay file to mcuboot, to enable CONFIG_FLASH_SIMULATOR=y in mcuboot.